### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by meet9111**
I have added everything

### CODE REVIEW

**Comment by meet9111** [comment]
I have added everything

File: index.html [hunk 1]
```diff
@@ -23,10 +23,22 @@
         <div class="page_footer">
            <div class="Side_By_Side_2">
                <div class="sec1"></div>
-               <div class="sec2"></div>
+               <div class="sec2">
+                   <p id="sec1text">"Made With Love By Members of UTC"</p>
```
Comment (Leo-Malay) [inline index.html:27]: Let Class name be --> main_text

File: index.html [hunk 2]
```diff
@@ -23,10 +23,22 @@
         <div class="page_footer">
            <div class="Side_By_Side_2">
                <div class="sec1"></div>
-               <div class="sec2"></div>
+               <div class="sec2">
+                   <p id="sec1text">"Made With Love By Members of UTC"</p>
+               </div>
+           </div>
+           <div class="navi_link">
+            <ul id="navi_list">
```
Comment (Leo-Malay) [inline index.html:31]: No id required

File: index.html [hunk 3]
```diff
@@ -23,10 +23,22 @@
...
            <div class="Side_By_Side_2">
                <div class="sec1"></div>
-               <div class="sec2"></div>
+               <div class="sec2">
+                   <p id="sec1text">"Made With Love By Members of UTC"</p>
+               </div>
+           </div>
+           <div class="navi_link">
+            <ul id="navi_list">
+                <li>Home</li>
```
Comment (Leo-Malay) [inline index.html:32]: add <a href="./###.html"> in every li tag

File: index.html [hunk 4]
```diff
@@ -23,10 +23,22 @@
...
+                <li>Aboutus</li>
+                <li>Project</li>
+                <li>Teams</li>
+                <li>Contact</li>
+              </ul>  
+           </div>
+           <div class="footer_text">
+               <p id="footer_text1">All Right Reserved 2020</p>
```
Comment (Leo-Malay) [inline index.html:40]: Add <br> tag in between

Comment (Leo-Malay) [reply index.html:40]: id ---> copyright_text

**Review by Leo-Malay — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (Leo-Malay) [reply index.html:32]: <a> tag

Comment (Leo-Malay) [reply index.html:32]: a

**Review by Leo-Malay — APPROVED** [review]
Done

**Pull request merged**